settings.gradle

```diff
@@ -6,12 +6,14 @@ include ':logins-library'
 include ':places-library'
 include ':rustlog-library'
 include ':as-support-library'
+include ':push-library'
```

Comment:
You should probably file a follow up if one doesn't exist for getting this into the megazords.